leveranciers.md

```diff
@@ -96,7 +96,7 @@ zelfs wenselijk om isolatie tussen projecten te borgen.
 Afstemmen welke zaaktypen nodig zijn zal waarschijnlijk met Functioneel Beheer gebeuren.
 
 ## Documenten API
-
+[nog aanpassen https://github.com/GemeenteUtrecht/ZGW/issues/1142 ]
 Momenteel staat de Documenten API gehost op https://drc.utrechtproeftuin.nl/. Dit is een
```

Comment:
Dit moet zijn: https://alfresco.cg-intern.ont.utrecht.nl/alfresco/service/drc/v1/

Reply:
De URL naar de documentatie ervan moeten we inderdaad nog toevoegen. Of verwijzen naar de documentatie van de OpenZaak Documenten API, aangezien die hetzelfde werkt.